pedro/src/main/kotlin/dev/nextftc/extensions/pedro/Turn.kt

```diff
@@ -0,0 +1,15 @@
+package dev.nextftc.extensions.pedro
+
+import dev.nextftc.core.commands.Command
+import dev.nextftc.core.units.Angle
+import dev.nextftc.core.units.abs
+
+class Turn(private val angle: Angle) : Command() {
```

Comment:
It might be useful to name this like `"Turn($angle)"`.
```suggestion
class Turn(private val angle: Angle) : Command() {
  init {
    named("Turn($angle)"
  }
```

Reply:
Right, I forgot names existed. This suggestion is missing a closing parenthesis, but I can add this tomorrow.